vb-fe/.eslintrc.json

```diff
@@ -56,7 +56,7 @@
         }
       ],
       "import/no-unresolved": "error",
-      "indent": [2, "tab"],
+      "indent": [4, "tab"],
```

Comment:
no way jose we have beef here

Reply:
my shit wasn't compiling!!!! that was the only fix lol!!!!!

Reply:
No way lmao, how weird.  All good with me, as long as the BE is on 2